dotcom-rendering/scripts/jsonSchema/schema.mjs

```diff
@@ -62,6 +63,10 @@ const schemas = [
 		typeName: 'FEFootballDataPage',
 		file: `${root}/src/model/fe-football-data-page-schema.json`,
 	},
+	{
+		typeName: 'FECricketMatchPage',
+		file: `${root}/src/model/fe-cricket-match-page-schema.json`,
```

Comment:
As above, can we put this in the `frontend` directory and name it after the core type it exports?

```suggestion
		file: `${root}/src/frontend/schemas/feCricketMatchPage.json`,
```